pkg/token/token.go

```diff
@@ -20,6 +20,7 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
```

Comment:
nit, import is out of order w/ other aws-sdk-go imports

Reply:
nice catch :D